.kokoro/run_samples_tests.sh

```diff
@@ -16,42 +16,19 @@
 # `-e` enables the script to automatically fail when a command fails
 # `-o pipefail` sets the exit code to the rightmost comment to exit with a non-zero
 set -eo pipefail
-# Enables `**` to include files nested inside sub-folders
-shopt -s globstar
 
-# Update `gcloud` and log versioning for debugging.
-gcloud components install beta --quiet
-gcloud components update --quiet
-echo "********** GCLOUD INFO ***********"
-gcloud -v
 echo "********** MAVEN INFO  ***********"
 mvn -v
 
 # Setup required env variables
 source ${KOKORO_GFILE_DIR}/bigquery_secrets.txt
 echo "********** Successfully Set All Environment Variables **********"
 
-# Activate service account
-gcloud auth activate-service-account \
```

Comment:
I believe you should still use this to make sure you are using the correct service account and project.